examples/tutorials/online_asr_tutorial.py

```diff
@@ -222,9 +223,10 @@ def run_inference(num_iter=100):
         segment = cacher(chunk[:, 0])
         features, length = feature_extractor(segment)
         hypos, state = decoder.infer(features, length, 10, state=state, hypothesis=hypothesis)
-        hypothesis = hypos[0]
-        transcript = token_processor(hypothesis[0], lstrip=False)
-        print(transcript, end="", flush=True)
+        hypothesis = hypos
+        transcript = token_processor(hypos[0][0], lstrip=False)
+        os.system('cls' if os.name == 'nt' else 'clear')
```

Comment:
is this necessary?

Reply:
This is because the transcript is long and single line refresh won't work.

Reply:
Can you show a screen shot of what issue happens due to long line?
I don't think system call belongs here.

Reply:
This example is a transcription of a very long audio, which spans multiple lines on the screen. Since the transcript now updates every step but keeps all the prior history, if we didn't clear the screen, we end up with repetitions of the same line. Using carriage return on print works, if we are refreshing the same line. Here we want to refresh several previous print lines.

Reply:
@lakshmi-speak it seems to work fine without the system call. can you remove it? the rest of the pr looks good — we can merge it after you make the change.

Reply:
@hwangjeff ready to merge!